gems/smithy-client/lib/smithy-client/signers/http_api_key.rb

```diff
@@ -5,7 +5,12 @@ module Client
     module Signers
       # A signer that signs requests using the HTTP API Key Auth scheme.
       class HttpApiKey < Signer
-        def sign(request:, identity:, properties:)
+        # rubocop:disable Metrics/AbcSize
+        def sign(context)
```

Comment:
Rather than enabling and disabling this specific cop on this method. You could:
```
def sign(context) # rubocop:disable Metrics/AbcSize
```
Then, this disable will be limited to that method. Matt had this in his PR and it was neat!